test/metabase/upload_test.clj

```diff
@@ -299,7 +340,7 @@
                     table))
             (is (=? {:name          #"(?i)id"
                      :semantic_type :type/PK
-                     :base_type     :type/BigInteger}
+                     :base_type     :type/Integer}
```

Comment:
We should have a test where the `id` column integers outside the range of an integer but still in the range of a bigint